examples/cookbook/design/themes/lib/main.dart

```diff
@@ -11,30 +14,39 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     const appName = 'Custom Themes';
 
-    // #docregion MaterialApp
     return MaterialApp(
       title: appName,
       theme: ThemeData(
-        // Define the default brightness and colors.
-        brightness: Brightness.dark,
-        primaryColor: Colors.lightBlue[800],
+        useMaterial3: true,
 
-        // Define the default font family.
-        fontFamily: 'Georgia',
+        // Define the default brightness and colors.
+        colorScheme: ColorScheme.fromSwatch(
+          primarySwatch: Colors.purple,
+          // TRY THIS: Change to "Brightness.light"
+          //           and see that all colors change
+          //           to better contrast a light background.
+          brightness: Brightness.dark,
+        ),
 
         // Define the default `TextTheme`. Use this to specify the default
         // text styling for headlines, titles, bodies of text, and more.
-        textTheme: const TextTheme(
-          displayLarge: TextStyle(fontSize: 72, fontWeight: FontWeight.bold),
-          titleLarge: TextStyle(fontSize: 36, fontStyle: FontStyle.italic),
-          bodyMedium: TextStyle(fontSize: 14, fontFamily: 'Hind'),
+        textTheme: TextTheme(
+          displayLarge:
+              const TextStyle(fontSize: 72, fontWeight: FontWeight.bold),
+          // TRY THIS: Change one of the GoogleFonts
```

Comment:
@Piinks suggested it. Thought it was a great idea.

Reply:
I've seen it start cropping up in samples in the API docs. Since dartpad in the API docs makes them interactive, 'try this's hopefully give folks more hands on learning. 😄